samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -9,20 +9,20 @@
     "Label": "Completion",
     "AIService": "[Enter 'AzureOpenAI', or 'OpenAI']",
     "DeploymentOrModelId": "[Enter the Azure OpenAI deployment name or OpenAI model, e.g. 'text-davinci-003']",
-    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]"
+    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]",
     /* Use user secrets or Key Vault - Putting secrets in a plain-text file
        is a security risk and should be done as a last resort only. */
-    //"Key": "[DO NOT CHECK IN - Key to your (Azure) OpenAI instance.]"
+    "Key": "[DO NOT CHECK IN - Key to your (Azure) OpenAI instance.]"
```

Comment:
Can you revert this file?  Before, it was how you made it but Devis commented on how easy it was for people to check in secrets, so we made it more work (minimally) for people to use Key from the file itself...

Reply:
Oh I see, I'll change it. Is the expectation to set this key using the following command instead?
dotnet user-secrets set "CompletionConfig:Key" "MY_SECRET_VALUE"